drivers/ethernet/eth_e1000.c

```diff
@@ -9,7 +9,7 @@
 #include <logging/log.h>
```

Comment:
The change is good, the commit message has incorrect information as it is minimal libc that is missing unistd.h (newlibc has it). Can you submit a version that fixes that info?